ci/travis_install_toolchain.sh

```diff
@@ -26,6 +26,7 @@ if [ ! -e $CPP_TOOLCHAIN ]; then
     conda create -y -q -p $CPP_TOOLCHAIN python=2.7 \
         nomkl \
         boost-cpp \
+        libprotobuf \
```

Comment:
Do we also need to install `protobuf`? Are we assuming that `libprotobuf` and the `protoc` called by the orc build step are compatible?

Reply:
libprotobuf includes protoc

Reply:
Oh okay, cool.

Reply:
I guess we are assuming that orc's protoc and ours are compatible? I could be missing something obvious here.

Reply:
The ORC build will just use this one; before this patch we were building protobuf ourselves from source in https://github.com/apache/arrow/blob/master/cpp/cmake_modules/ThirdpartyToolchain.cmake#L882

This does upgrade protocol buffers from 2.6.0 to 3.5.1, but I think it's forward compatible (might be wrong). If not we may need to add a protobuf 2.x package to conda-forge